soc/arm/st_stm32/stm32f7/Kconfig.defconfig.series

```diff
@@ -52,4 +52,11 @@ config ENTROPY_STM32_RNG
 
 endif # ENTROPY_GENERATOR
 
+if DMA_STM32
+
+config DMA_STM32_V1
+	default y
+
+endif # DMA
```

Comment:
This is the same as this btw:

```
config DMA_STM32_V1
	default y if DMA_STM32
```

Reply:
Ok, that's better

Reply:
I think I would still use the style I was using, all other driver kconfig symbols in these files uses that style, should be consistent with them.